.pre-commit-config.yaml

```diff
@@ -29,3 +29,10 @@ repos:
         stages: [manual]
         args: ["--py310-plus"]
         exclude: ^tests/.*snapshots/
+  - repo: local
+    hooks:
+      - id: gen-docs
+        name: gen-docs
+        entry: ./gen_docs.sh
```

Comment:
Okay, I get the point with the `justfile` now... I don't know if it is possible to run `just` here...

Reply:
I think it is not possible
https://pre-commit.com/#supported-languages

Reply:
It would have to be a bash script containing `just generate-docs`